sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/ImplementIDisposableCorrectly.cs

```diff
@@ -100,9 +100,9 @@ public IEnumerable<SecondaryLocation> GetIssueLocations()
                                 + " and override the base class 'Dispose' implementation instead.");
                     }
 
-                    if (FindMethodDeclarationsRecursive(classSymbol.BaseType, IsVirtualDisposeBool).Any())
+                    if (HasVirtualDisposeBoolImplementationRecursiveSearch(classSymbol.BaseType))

```

Comment:
I'm not sure about this name - should remove RecursiveSearch?

Reply:
I think you should remove it, yes :)